packages/cashier/src/pages/account-transfer/account-transfer-receipt/account-transfer-receipt.tsx

```diff
@@ -64,13 +64,10 @@ const AccountTransferReceipt = observer(({ onClose, history }: TAccountTransferR
         } else {
             // if the account transferred to is a Deriv MT5 account that can't be switched to, switch to from account instead
             // otherwise switch to the account transferred to
-            setShouldSwitchAccount();
+            setShouldSwitchAccount(true);
             setSwitchTo(selected_to.is_mt ? selected_from : selected_to);
             toggleSwitchAlert();
         }
-        // close modal only when the user try to transfer money from traders-hub, not from cashier
-        // because in cashier this component is not a modal
-        if (is_from_outside_cashier) onClose?.();
```

Comment:
I deleted the strings because I assume that it is related to line `73`. Apparently, according to the QA the modal should show and redirect to the reports section especially when Traders Hub had become the default UI.

Reply:
@aizad-deriv Do you think it's ok when we goes back to trader's hub from 'transaction details' this transfer modal open? I think the modal must be closed, am I wrong?

Reply:
True you made a solid point I will incorporate the changes for the next commit. 😃 👍🏼

Reply:
@sergei-deriv alright I do agree with what you are saying I'll notify the QA and get on working on this. 🙏🏼